sdk/tables/Azure.Data.Tables/tests/TableClientLiveTests.cs

```diff
@@ -110,9 +111,368 @@ public async Task InsertedEntitiesCanBeQueriedWithFilters()
 
                 // Query the entities with a filter specifying that to RowKey value must be greater than or equal to '10'.
 
-                entityResults = (await client.QueryAsync(filter: $"PartitionKey eq '{partitionKeyValue}' and RowKey ge '10'").ToEnumerableAsync().ConfigureAwait(false)).ToList();
+                entityResults = (await client.QueryAsync(filter: $"PartitionKey eq '{partitionKeyValue}' and RowKey gt '10'").ToEnumerableAsync().ConfigureAwait(false)).ToList();
+
+                Assert.That(entityResults.Count, Is.EqualTo(10), "The entity result count should be 10");
+            }
+            finally
+            {
+                if (doCleanup)
+                {
+                    await service.DeleteTableAsync(tableName);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Validates the functionality of the TableClient.
+        /// </summary>
+        [Test]
+        public async Task EntitiyCanBeUpserted()
+        {
+            string tableName = $"testtable{Recording.GenerateId()}";
+            const string partitionKeyValue = "somPartition";
+            const string rowKeyValue = "1";
+            const string propertyName = "SomeStringProperty";
+            const string originalValue = "This is the original";
+            const string updatedValue = "This is new and improved!";
+            bool doCleanup = false;
+            TableServiceClient service = CreateTableServiceClient();
```

Comment:
If all tests would need a table we might think about having a SetUp and TearDown methods that would manage that.

Reply:
I considered that for the whole class, but doing it per test might be better, as you suggested.

Reply:
I'm going to punt this to a future PR